apple-pie/styles/style.css

```
@@ -0,0 +1,95 @@
+.container{
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    font-family: 'Roboto', sans-serif;
+    height: 1000px;
+    width: 1520px;
+
+}
+
+.applepic{
+    height: 1000px;
+    width: 1450px;
+   
+}
+
+.applepie{
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    color: white;
+    font-size: 100px;
+}
+
+.box1{
+    margin-top: 50px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  
+}
+
+.text1{
+    font-family: 'Roboto', sans-serif; 
+    width: 700px;
+    font-size: 23px;
+}
+
+.time{
+    width: 400px;
+}
+
+.linea1{
+    width: 700px;
+    height: 2px;
+    background-color: gray;
+    margin-bottom: 30px;
+}
+
+.cooking{
```

Otra opción al comentario de arriba 
```suggestion
.cooking, .time{
```